opencti-platform/opencti-graphql/src/connector/importCsv/importCsv-connector.ts

```diff
@@ -36,6 +40,35 @@ const initImportCsvConnector = () => {
     rabbitMqConnection = conn;
   };
 
+  const generateBundle = async (context: AuthContext, csvMapper: CsvMapperParsed, messageParsed: any, entity: BasicStoreBase | undefined, csvLines: string[]) => {
+    const workId = messageParsed.internal.work_id;
+    const applicantId = messageParsed.internal.applicant_id;
+    const applicantUser = await resolveUserByIdFromCache(context, applicantId) as AuthUser;
+    const bundle = await bundleProcess(context, applicantUser, csvLines, csvMapper, { entity });
+    const validateBeforeImport = connectorConfig.config.validate_before_import;
+    if (validateBeforeImport) {
+      await updateExpectationsNumber(context, applicantUser, workId, 1);
+      const contentStream = Readable.from([JSON.stringify(bundle, null, '  ')]);
+      const file = {
+        createReadStream: () => contentStream,
+        filename: `${workId}.json`,
+        mimetype: 'application/json',
+      };
+      await uploadToStorage(context, applicantUser, 'import/pending', file, { entity });
```

Comment:
so it will upload an analyst workbench file with a bundle for the current lines that are being parsed ? does it append the file or replace it only ? From what I read it looks like it will replace it.

Reply:
Really good point, i need to dig in into this